Vitals/AppDelegate.swift

```diff
@@ -10,25 +10,18 @@ import SwiftUI
 
 @main
 class AppDelegate: NSObject, NSApplicationDelegate {
-    var popover = NSPopover.init()
     var statusBar: StatusBarController?
     let processMonitor = ProcessMonitor()
 
     func applicationDidFinishLaunching(_ aNotification: Notification) {
         processMonitor.start()
         
         // Create the SwiftUI view that provides the contents
-        let viewModel = ContentViewModel(monitor: processMonitor, contentVisible: false)
+        let viewModel = ContentViewModel(monitor: processMonitor, contentVisible: true)
         let contentView = ContentView(viewModel: viewModel)
-
-        // Note: we _don't_ want to make the popover's behaviour "transient" as we handle all the click
-        // detection ourselves in the StatusBarController, which gives us more control
-        popover.contentSize = NSSize(width: 400, height: 360)
-        // Put the SwiftUI content inside the popover
-        popover.contentViewController = NSHostingController(rootView: contentView)
-        popover.animates = false
-
-        statusBar = StatusBarController.init(popover, contentViewModel: viewModel)
+        
+        let v = NSHostingView(rootView: contentView)
+        statusBar = StatusBarController.init(contentViewModel: viewModel, contentView: v)
```

Comment:
Nit: I know the explicit `.init()` is a holdover from previous commits, but could it be removed for consistency?
Also, although it literally gets used the next line, I think the `NSHostingView` instance could use a more expressive name.
```suggestion
        
        let hostingView = NSHostingView(rootView: contentView)
        statusBar = StatusBarController(contentViewModel: viewModel, contentView: hostingView)
```